src/ResourceManager/RecoveryServices.SiteRecovery/Commands.RecoveryServices.SiteRecovery/help/New-AzureRmRecoveryServicesAsrNetworkMapping.md

```diff
@@ -36,10 +43,31 @@ The **New-AzureRmRecoveryServicesAsrNetworkMapping** cmdlet starts the operation
 PS C:\> $currentJob = New-AzureRmRecoveryServicesAsrNetworkMapping -Name $NetworkMapName -PrimaryNetwork $PrimaryNetwork -RecoveryNetwork $RecoveryNetwork
 ```
 
+### Example 2
+```
+PS C:\> $currentJob = New-AzureRmRecoveryServicesAsrNetworkMapping -AzureToAzure -Name "mappingName" -PrimaryFabric $AzureFabric
+ -PrimaryAzureNetworkId $AzureNetworkId -RecoveryFabric $RecoveryAzureFabric -RecoveryAzureNetworkId $RecoveryNetworkId
+```
+
 Starts the network mapping creation operation using the specified name, primary and recovery networks, and returns an ASR job to track the operation.
```

Comment:
@viverm same comment about the description being moved from the first example to the example added. Also, the example provided won't work if a user were to try and copy-paste the contents since you broke the command into separate lines with adding a trailing backtick ( ` ) at the end of the first line to signal you are continuing to another line

Reply:
comment taken